nflow-netty/src/test/java/io/nflow/netty/StartNflowTest.java

```diff
@@ -28,18 +37,29 @@ public void startNflowNetty() throws Exception {
         .registerSpringClasspathPropertySource("external.properties")
         .registerSpringApplicationListener(testListener);
     Map<String, Object> properties = new HashMap<>();
-    properties.put("nflow.db.create_on_startup", false);
+    String restApiPrefix = "nflow/api";
+    properties.put("nflow.db.create_on_startup", true);
     properties.put("nflow.autostart", false);
-    properties.put("nflow.autoinit", false);
+    properties.put("nflow.autoinit", true);
+    properties.put("nflow.rest.path.prefix", restApiPrefix);
+
     ApplicationContext ctx = startNflow.startNetty(7500, "local", "", properties);
 
     assertNotNull(testListener.applicationContextEvent);
     assertEquals("7500", ctx.getEnvironment().getProperty("port"));
     assertEquals("local", ctx.getEnvironment().getProperty("env"));
     assertEquals("externallyDefinedExecutorGroup", ctx.getEnvironment().getProperty("nflow.executor.group"));
-    assertEquals("false", ctx.getEnvironment().getProperty("nflow.db.create_on_startup"));
+    assertEquals("true", ctx.getEnvironment().getProperty("nflow.db.create_on_startup"));
     assertEquals("false", ctx.getEnvironment().getProperty("nflow.autostart"));
-    assertEquals("false", ctx.getEnvironment().getProperty("nflow.autoinit"));
+    assertEquals("true", ctx.getEnvironment().getProperty("nflow.autoinit"));
+
+    // Smoke test here also the netty REST API by fetching workflow definitions
```

Comment:
Extract this block to a method (with a good name, the comment may not be needed anymore).
Also consider using static imports at least for well-named constants.